idseq/uploader.py

```diff
@@ -137,6 +137,10 @@ def upload(
             'http://%s/samples/%d.json' %
             (url, data['id']), data=json.dumps(update), headers=headers)
 
+        requests.put(
+            'http://%s/samples/%d/kickoff_pipeline' % (url, data['id']),
+            headers={ 'X-User-Email': email, 'X-User-Token': token })
+
```

Comment:
you will have to make the kickoff_pipeline function token_authenticated and turn off csrf

Reply:
in idseq-web

Reply:
Ah! Thanks, I turned off csrf but didn't make it token_authenticated, so that;'s probably why it didn't work